src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -161,10 +161,9 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 	private _ctxMenuWidgetVisible: IContextKey<boolean>;
 	private viewItems: ICodeActionMenuItem[] = [];
 	private focusedEnabledItem: number | undefined;
-	private currSelectedItem: number = 0;
+	private currSelectedItem: number | null = 0;
```

Comment:
Should this default to undefined instead of `0`?

Also prefer `undefined` over `null`